app/bundles/EmailBundle/Helper/FormSubmitHelper.php

```diff
@@ -63,9 +63,10 @@ public static function sendEmail($tokens, Action $action, MauticFactory $factory
             } elseif (isset($currentLead)) {
                 if (isset($leadFields['email'])) {
                     $options = [
-                        'source'    => ['form', $form->getId()],
-                        'tokens'    => $tokens,
-                        'ignoreDNC' => true,
+                        'source'      => ['form', $form->getId()],
+                        'tokens'      => $tokens,
+                        'ignoreDNC'   => true,
+                        'immediately' => ($factory->getParameter('mailer_spool_type') == 'file' && !empty($properties['immediately'])) ? true : false,
```

Comment:
Can you include CoreParameterHelper like you did it in EmailSendType? Let's not use the MauticFactory in the new code. And again, use `===`

Reply:
But this is static function. I cannot use $this->coreParametersHelper, can I?